### PR TITLE
fixed with sidebar overlapping and designing issues

### PR DESCRIPTION
**Pull request by dilpreet77**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by ravinderwalkover — APPROVED** [review]

File: apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.html [hunk 1]
```diff
@@ -5,8 +5,8 @@
                 {{ commonLocaleData?.app_buy_new_plan }}
             </button>
         </div>
-        <mat-grid-list cols="16" rowHeight="40px" gutterSize="0px" class="responsive-grid-list">
-            <mat-grid-tile colspan="4" rowspan="1" class="responsive-grid-title-left-top">
+        <mat-grid-list cols="16" rowHeight="40px" gutterSize="0px">
+            <mat-grid-tile colspan="{{ menuOneWidth }}" rowspan="1">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.html:9]: Consider using `[attr.colspan]="menuOneWidth"` through out this file where colspan is used

File: apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.html [hunk 2]
```diff
@@ -21,11 +21,15 @@
                     </mat-form-field>
                 </div>
             </mat-grid-tile>
-            <mat-grid-tile colspan="12" class="responsive-grid-title-right">
+            <mat-grid-tile colspan="{{ menuTwoWidth }}">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.html:24]: Consider using `[attr.colspan]` through out this file where colspan is used

File: apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.html [hunk 3]
```diff
@@ -88,10 +92,10 @@
                     >
                 </div>
             </mat-grid-tile>
-            <mat-grid-tile colspan="4" rowspan="15" class="responsive-grid-title-left">
+            <mat-grid-tile colspan="{{ sideBarBoxWidth }}" rowspan="{{ sideBarBoxLength }}">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.html:95]: Consider using `[attr.colspan]` & `[attr. rowspan]` through out this file where either colspan or rowspan is used

File: apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.html [hunk 4]
```diff
@@ -88,10 +92,10 @@
...
                 <div class="sidebar" *ngIf="!showLoader">
                     <ng-container *ngFor="let subscription of subscriptions; let i = index">
-                        <mat-grid-list cols="2" rowHeight="150px" gutterSize="0px" class="sidebar-box">
+                        <mat-grid-list cols="2" rowHeight="{{ rowLength }}" gutterSize="0px" class="sidebar-box">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.html:98]: Consider using `[rowHeight]="rowLength"`

File: apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.ts [hunk 5]
```diff
@@ -49,6 +49,18 @@ export class SubscriptionComponent implements OnInit, OnDestroy, OnChanges {
     @ViewChild(MatAccordion) accordion: MatAccordion;
     /** This will use for move company in to another company  */
     @ViewChild("moveCompany", { static: false }) public moveCompany: any;
+    /** This will change the search bar width dynamically */
+    public menuOneWidth: Number = 4;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.ts:53]: Consider using primitive type `number` in place of `Number` through out this file

File: apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.ts [hunk 6]
```diff
@@ -147,6 +161,46 @@ export class SubscriptionComponent implements OnInit, OnDestroy, OnChanges {
         /** This will use for image format */
         this.imgPath = (isElectron || isCordova) ? 'assets/images/' : AppUrl + APP_FOLDER + 'assets/images/';
 
+        this.store.pipe(select(state => state.session.activeCompany), takeUntil(this.destroyed$)).subscribe(activeCompany => {
+            if (activeCompany) {
+                this.activeCompany = activeCompany;
+            }
+        });
+
+        // /** This will use for responsive */
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.ts:170]: Improper comment

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**